CMakeLists.txt

```diff
@@ -34,7 +31,20 @@ include(blt/SetupBLT.cmake)
 # Define RAJA PERFSUITE settings...
 #
 
+#cmake_dependent_option(RAJA_PERFSUITE_ENABLE_TESTS "Enable RAJA Perf Suite Tests" On "ENABLE_TESTS" Off)
+
+if (ENABLE_TESTS)
+
+  set(RAJA_PERFSUITE_ENABLE_TESTS On CACHE BOOL "Enable testing for RAJA Perf Suite")
```

Comment:
Why is this done like this?

Reply:
I wanted to turn off camp and RAJA tests because they take too long to build. I added a new variable to emphasize the point. It could be done other ways, I suppose. Is it confusing?

Reply:
I mean why not use the commented out cmake_dependent_option?

Reply:
Ah. I think I tried that and it didn't work for me. It may have been due to other breakage I was dealing with.

Reply:
It would be good to use the dependent option if possible.

Reply:
If I do that, I would have to set both ENABLE_TESTS and RAJA_PERFSUITE_ENABLE_TESTS to On? Or, am I mis-understanding?

Reply:
The dependent option would set `RAJA_PERFSUITE_ENABLE_TESTS` to `On` if `ENABLE_TESTS` is On - so you only have to set `ENABLE_TESTS`.

Reply:
Thanks. Will try.